packages/itwin/grouping-mapping-widget/src/components/Properties/GroupProperties/GroupPropertyTable.tsx

```diff
@@ -12,7 +25,6 @@ import { PropertyTable } from "../PropertyTable";
 import { useGroupingMappingApiConfig } from "../../context/GroupingApiConfigContext";
 import { useMutation, useQueryClient } from "@tanstack/react-query";
 import { usePropertiesClient } from "../../context/PropertiesClientContext";
-
 export interface GroupPropertyTableProps {
```

Comment:
Missing line

Reply:
Fixed with prettier.

Reply:
Still no line between the interface and import statement.

Reply:
I though prettier did for me lol, extra line added now.